docs/config-api.md

```diff
@@ -175,6 +175,13 @@ System.config({
 });
 ```
 
+* `format`: Sets in what format the module is loaded. See [Module Formats](https://github.com/systemjs/systemjs/blob/master/docs/module-formats.md).
+* `exports`: For `global` format, when automatic detection of exports is not enough, a custom exports meta value can be set. It tells the loader what global name to read, this is then used as the module's export value. More info: [Exports](https://github.com/systemjs/systemjs/blob/master/docs/module-formats.md#exports)
+* `deps`: Dependencies to load before this module. Goes through regular paths and map normalization. More info: [Shim Dependencies](https://github.com/systemjs/systemjs/blob/master/docs/module-formats.md#shim-dependencies). Only supported for the `cjs`, `amd` and `global` formats.
+* `globals`: Allows creating globals and pointing them to `cjs` or `global` packages, this let's you guarantee certain globals exist. These globals will only be available while the package described by the meta path is executing. They will be reverted to whatever they where before after execution, if they didn't exist they're removed. This doesn't influence the globals that might already be generated by the referenced package. Referenced packages automatically becomes dependencies. More info: [Custom Globals](https://github.com/systemjs/systemjs/blob/master/docs/module-formats.md#custom-globals).
```

Comment:
A map of global names to module names that should be defined only for the execution of this module. Allows legacy code that expects certain globals to be present to be supported via configuration.